quickwit/quickwit-telemetry/src/lib.rs

```diff
@@ -52,7 +52,7 @@ fn get_telemetry_sender_singleton() -> &'static TelemetrySender {
 /// We voluntarily use an enum here to make it easier for reader
 /// to audit the type of information that is send home.
 pub async fn send_telemetry_event(event: TelemetryEvent) {
-    get_telemetry_sender_singleton().send(event).await
+    TELEMETRY_SENDER.get().map(|sender| sender.send(event));
```

Comment:
```suggestion
    if let Some(telemetry_sender) = TELEMETRY.get() {
        sender.send(event);
    }
```

It is really a if here. We don't return an expression.